src/Molecule.py

```diff
@@ -133,7 +133,10 @@ def drawMoleculeAsByteStream(self, orient_with_scaffold=False):
             # assignment is sufficient
             _ = AllChem.GenerateDepictionMatching2DStructure(drawn_mol,
                                                              orient_scaffold)
-        img = Chem.Draw.MolToImage(drawn_mol)
+        if size is not None:
```

Comment:
I don't understand what this is doing. Maybe you should comment it?

Reply:
it checks if size has been passed as a parameter to the function, so only executes when its there. Function